rpmbuild/copr_rpmbuild/helpers.py

```diff
@@ -428,14 +428,22 @@ def exclusivearch(self):
         """
         Evaluated %{exclusivearch} as a list
         """
-        return self.safe_attr("exclusivearch").split()
+        values = self.safe_attr("exclusivearch").split()
+        for macro in values:
+            if macro.startswith("%"):
+                log.warning("Unknown macro in ExclusiveArch: %s", macro)
+        return values
```

Comment:
I'm not complaining, just curious ... When we are doing this already, why do we have to report back to backend && frontend useless set of macros (some unexpanded)?  Wouldn't it be better to return just empty list?

Reply:
Updpated, PTAL